metals/src/main/scala/scala/meta/internal/metals/BloopInstall.scala

```diff
@@ -214,6 +214,7 @@ object BloopInstall {
   private class ProcessHandler() extends NuAbstractProcessHandler {
     var response: Option[CompletableFuture[_]] = None
     val completeProcess = Promise[BloopInstallResult]()
+    var output = ""
```

Comment:
It looks like both of the input channels are using the same "queue"? Is there no contention between them? Does the `onStdout` and `onStderr` happen in the exact same order as in the process? Are they handled by the same thread underneath (i.e. the Process class in scala library there is a thread for each of them so they might trigger those methods in any order they want). If there are two threads handling `stderr` and `stdout` then we either need some means of synchronization on the `log` method or even better two queues. Otherwise those would  probably get intertwined. 
But if everything here is safe, then please add a comment ;)

Reply:
Separated the different outputs - thanks! Haven't noticed it.